### ISSUE TITLE
[QRADAR] API Error

### ISSUE THREAD

**Issue by cattleindigo** [OpenCTI-Platform/connectors#2722]
## Description

Setting up Qradar connector with OpenCTI and seems to have a few issues with the sent information. Error code 422: The request was well-formed but was unable to be followed due to semantic errors.

## Environment

1. OS Ubuntu 16.4
2. OpenCTI version: 6.2.12
3. OpenCTI client: docker deployment
4. Other environment details: 

## Reproducible Steps

Steps to create the smallest reproducible scenario:
1. Pulled opencti/connector-qradar:6.2.12
2. Configured yml file as in github
3. Update the stack

## Expected Output

Creation of reference sets in qradar

## Actual Output

Error logs created for error 422
![Screenshot 2024-09-26 120006](https://github.com/user-attachments/assets/5c307e77-e1b3-4f5f-85db-35f3bdaf48de)
![Screenshot 2024-09-26 120040](https://github.com/user-attachments/assets/956919fe-2f3d-41cd-ae15-0255ea7ae7f8)

**Comment by romain-filigran** [comment]
Hello @cattleindigo : The QRadar connector was designed to push entities of type "indicator" only in QRadar. In your logs, it seems that you are also trying to push "Report" entity. Can you reconfigure your stream definition to include only entity of type "indicator" to prevent this error ?

**Comment by cattleindigo** [comment]
Done, this is the current error log
![Screenshot 2024-09-30 092413](https://github.com/user-attachments/assets/8cfe9ad0-a0f4-4d8c-a3c1-c577e2be6c81)
![Screenshot 2024-09-30 092440](https://github.com/user-attachments/assets/943551d8-10f0-4513-82e4-0b42164ab016)

**Comment by romain-filigran** [comment]
This error is different and indicate that the connector is not able to communicate with your OpenCTI live stream. Does your connector and your OpenCTI are running in the same docker context ?

**Comment by cattleindigo** [comment]
Yes, it's a docker deployment with portainer used to deploy our OpenCTI stack. There isn't anything else running on it.

**Comment by nino-filigran** [comment]
@cattleindigo do you still have an issue?

**Comment by romain-filigran** [comment]
This issue has been open for a long time without recent activity. We are closing it to keep the repository clean. If the problem persists with a recent version of the connector, please feel free to open a new issue with updated details (connector version, OpenCTI version, logs, and steps to reproduce). Thank you!
